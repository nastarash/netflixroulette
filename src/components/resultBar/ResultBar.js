import React from 'react';

class ResultBar extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return <div />;
  }
}

export default ResultBar;
