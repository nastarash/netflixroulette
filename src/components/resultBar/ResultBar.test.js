import React from 'react';
import renderer from 'react-test-renderer';
import ResultBar from './ResultBar';

it('render correctly', () => {
  const AppComponent = renderer.create(<ResultBar />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
