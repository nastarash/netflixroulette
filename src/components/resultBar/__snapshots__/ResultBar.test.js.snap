// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
   
  0
   movies found
</div>
`;
